manifests/metacontroller.yaml

```diff
@@ -43,8 +43,8 @@ spec:
     singular: controllerrevision
     kind: ControllerRevision
 ---
-apiVersion: apps/v1beta1
-kind: Deployment
+apiVersion: apps/v1
```

Comment:
Let's use `apps/v1beta2` for now so we maintain compatibility with k8s 1.8.

Reply:
noted!

Reply:
@enisoc Does Metacontroller still support v1.8? We will have GKE 1.11 GA rollout by mid-August (i.e. will not support v1.8 on GKE around a month later). Do we want to change the `apps/v1beta2` to `apps/v1` at that moment?

Reply:
I'd like to keep support for 1.8 until we have a compelling use case for relying on a feature that only exists in 1.9+.